blockstack/blockstack-app
# fix: inconsistent balances

> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/blockstack/ux/actions/runs/1113952644).

This PR fixes inconsistent balances shown when a user has locked STX. In addition, it fixes a validation error shown when using the `Max` button to send STX.

Issue #1488 

![image](https://user-images.githubusercontent.com/6493321/128028220-81c0e6bc-d32b-4443-990c-1bfe61fb07ba.png)

![image](https://user-images.githubusercontent.com/6493321/128028241-778727bd-d35a-4de4-b413-925298c7f443.png)

![image](https://user-images.githubusercontent.com/6493321/128028256-274e1a61-d738-4d14-94ba-887f9999d610.png)

Testing:
- Test with an account that has locked STX
- Make sure the STX balance is consistent everywhere
- It should show the available balance with locked STX subtracted out
- Make sure the `Max` button in the send form uses the correct available balance
- Make sure the validation warning shows the correct available balance

cc/ @aulneau @kyranjamie @fbwoolf
